docker-compose.yml

```diff
@@ -712,6 +714,9 @@ services:
       - NugetPackageDirectory=/project/${relativeNugetPackageDirectory:-packages}
       - artifacts=/project/${relativeArtifacts:-tracer/src/bin/artifacts}
       - framework=${framework:-netcoreapp3.1}
+      - Filter=${Filter:- } #optional
+      - SampleName=${SampleName:- } #optional
+      - SampleName #optional
```

Comment:
What exactly does this `SampleName` do?

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/4447/commits/988b42e0935c1d8f40c674ac84f3b6888bfd0be0